source/PlayerInfo.cpp

```diff
@@ -385,7 +385,11 @@ void PlayerInfo::AddChanges(list<DataNode> &changes)
 		// Recalculate what systems have been seen.
 		GameData::UpdateSystems();
 		seen.clear();
-		double jumpRange = (flagship && flagship->JumpRange()) ? flagship->JumpRange() : System::DEFAULT_NEIGHBOR_DISTANCE;
+		double jumpRange;
```

Comment:
Doesn't this give an uninitialized variable warning?

Reply:
I get 0 warnings on compiling this.

Reply:
Some compilers (probably including yours) perform default (zero) initialization for local variables, but I understood that the C++ specification does not require default initialization for local variables.
If other compilers don't perform default initialization, then this can lead to hard to find and hard to fix bugs when the code gets compiled with a different compiler, so please do perform default initialization of this local variable.